src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -790,6 +791,11 @@ namespace Google.Example.Library.V1
             )));
 
         /// <summary>
+        /// Settings used for long running operations.
+        /// </summary>
+        public OperationsSettings LongRunningSettings { get; set; }
```

Comment:
Any reason for this being called LongRunningSettings rather than OperationsSettings, or LongRunningOperationsSettings? While the latter is pretty wordy, I think it's clearer.